src/components/Image/Image.jsx

```diff
@@ -0,0 +1,36 @@
+import { PropTypes } from 'prop-types';
+import styled from '@emotion/styled';
+
+const Image = ({ src, alt, width, height, type, mode }) => {
+  const { PUBLIC_URL } = process.env;
+  const ProfileDefaultImage = `${PUBLIC_URL}/images/profile-image-default.png`;
+  const PostDefaultImage = `${PUBLIC_URL}/images/post-image-default.jpg`;
+
+  return (
+    <StyledImage
+      src={src || (type == 'profile' && ProfileDefaultImage) || PostDefaultImage}
+      alt={alt || '프로필 이미지'}
```

Comment:
```suggestion
      alt={alt}
```

보니까 Avatar에서 alt가 '프로필 이미지'로 들어가고 있어서 이렇게 하면 될 것 같습니다!